features/nationaliteithistorie.feature

```diff
@@ -3,128 +3,441 @@
-    | 0 | 0057               | -                                   | 1983-12-13            | 2014-06-01           | true                            | 404                  |
-    | 0 | 0100               | -                                   | 1989-03-01            | 1994-06-01           | true                            | 404                  |
-
-  Scenario: meerdere actuele en beëindigde nationaliteiten
-    Als de nationalteithistorie met burgerservicenummer 999990457 wordt geraadpleegd
-    Dan bevat het antwoord 5 voorkomens
-    En wordt de nationaliteithistorie teruggegeven in de volgorde en met waarden:
+  # in onderstaande scenario's staat ten behoeve van de provider implementatie in Gegeven achter de veldnaam tussen haakjes de veldcode zoals deze in de bron volgens LO GBA is geregistreerd
+
+  Rule: In het antwoord wordt indicatieNationaliteitBeeindigd opgenomen met de waarde true, wanneer in de actuele nationaliteit (categorie 04) GEEN nationaliteit (05.10) noch aanduiding bijzonder Nederlanderschap (65.10) is opgenomen, of wanneer in categorie 04 reden beëindigen nationaliteit (64.10) is opgenomen.
+
+    Scenario: actuele nationaliteit
+      Gegeven de ingeschreven persoon met burgerservicenummer 000009830 kent de volgende nationaliteiten:
+        | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
+        | 1      | 4         | 0001                  | 001                   |                          |                                    |                 | 19750707                        |
+      Als de nationaliteithistorie met burgerservicenummer 000009830 wordt geraadpleegd
+      Dan bevat het antwoord 1 item(s) voor nationaliteithistorie
+      En bevat het nationaliteithistorie-item met nationaliteit.code "0001" geen gegeven "indicatieNationaliteitBeeindigd"
+
+    Scenario: actueel bijzonder Nederlanderschap
+      Gegeven de ingeschreven persoon met burgerservicenummer 000009866 kent de volgende nationaliteiten:
+        | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
+        | 1      | 4         |                       | 310                   |                          | B                                  |                 | 19570115                        |
+      Als de nationaliteithistorie met burgerservicenummer 000009830 wordt geraadpleegd
+      Dan bevat het antwoord 1 item(s) voor nationaliteithistorie
+      En bevat het nationaliteithistorie-item met aanduidingBijzonderNederlanderschap "behandeld_als_nederlander" geen gegeven "indicatieNationaliteitBeeindigd"
+
+    Scenario: actuele en beëindigde nationaliteit
```

Comment:
Ik denk niet dat ik begrijp wat hier wordt beschreven. Rijen met dezelfde Stapel nummers horen bij elkaar? De volgorde van de rijen die horen bij een stapel lijkt te bepalen hoe de rijen moet worden geïnterpreteerd? Ik denk dat mijn moeite met het interpreteren van de scenario's komt doordat ik niet begrijp waarom al die andere categoriën in de Gegeven stap staan. Volgens de rule zou ik alleen naar 05.10, 64.10 en 65.10 moeten kijken. Als ik naar deze scenario kijk is dat dus niet voldoende. Of geldt er een andere rule voor een beëindigde nationaliteit

Reply:
Ik zie de rule voor beëindigde nationaliteit op regel 49. Ik denk dat het veel beter begrijpbaar wordt (als het mogelijk is) om actuele en beëindigde historie los van elkaar te beschrijven

Reply:
ik heb een toelichting op de registratie opgenomen. had dat eerst niet gedaan omdat dit vooral interessant is voor de provider, en die weet dat al (staat in LO GBA). Blijkbaar is een toelichting nodig om de feature en begrijpen, dus heb het nu wel toegevoegd